.github/workflows/get_charms_build_with_cache.yaml

```
@@ -0,0 +1,32 @@
+name: Get charms and build with cache (common)
```

suggestion to have the name like this just for better visibility in the GH UI
```suggestion
name: Build charms with cache
```
most importantly to see the `cache` part